examples/package.json

```diff
@@ -3,18 +3,19 @@
   "version": "0.0.0",
   "description": "Examples browser for the PlayCanvas Engine",
   "main": "index.js",
+  "type": "module",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "build": "cross-env ENVIRONMENT=production ENGINE_PATH=../build/playcanvas.js EXTRAS_PATH=../build/playcanvas-extras.js webpack --config webpack.config.js",
-    "webpack:watch": "webpack --config webpack.config.js --watch",
-    "build:watch": "cross-env ENGINE_PATH=../build/playcanvas.js EXTRAS_PATH=../build/playcanvas-extras.js concurrently --kill-others \"npm run webpack:watch\"",
-    "build:debug": "cross-env ENGINE_PATH=../build/playcanvas.dbg.js EXTRAS_PATH=../build/playcanvas-extras.js concurrently --kill-others \"npm run webpack:watch\"",
-    "build:profiler": "cross-env ENGINE_PATH=../build/playcanvas.prf.js EXTRAS_PATH=../build/playcanvas-extras.js concurrently --kill-others \"npm run webpack:watch\"",
+    "build": "npm run build:example:data && npm run build:directory && npm run build:iframes && cross-env rollup -c rollup.config.js",
+    "build:example:data": "node --es-module-specifier-resolution=node ./scripts/example-data.mjs",
+    "build:directory": "node --es-module-specifier-resolution=node ./scripts/example-directory/index.mjs",
+    "build:iframes": "cp -r ../build/ ./dist/build/ && cp ./lib/wasm-loader.js ./dist/build && cp ./node_modules/@playcanvas/observer/dist/index.js ./dist/build/playcanvas-observer.js && node --es-module-specifier-resolution=node ./scripts/iframe/index.mjs",
```

Comment:
This won't work on Windows.

Reply:
I've moved these into the script now. Hopefully that resolves any issues you have running it on windows.